module/attribute/src/Resources/config/services.yml

```diff
@@ -36,24 +36,15 @@ services:
             - ['set', ['Ergonode\Attribute\Domain\ValueObject\OptionValue\StringOption']]
             - ['set', ['Ergonode\Attribute\Domain\ValueObject\OptionValue\MultilingualOption']]
 
-    Ergonode\Attribute\Domain\Repository\AttributeRepositoryInterface: '@Ergonode\Attribute\Infrastructure\Persistence\Repository\DbalAttributeRepository'
-#    Ergonode\Attribute\Infrastructure\Persistence\Repository\Decorator\CacheAttributeRepositoryDecorator:
-#        decorates: Ergonode\Attribute\Domain\Repository\AttributeRepositoryInterface
-#        arguments:
-#            - '@Ergonode\Attribute\Infrastructure\Persistence\Repository\Decorator\CacheAttributeRepositoryDecorator.inner'
+    Ergonode\Attribute\Domain\Repository\AttributeRepositoryInterface:
+        alias: Ergonode\Attribute\Infrastructure\Persistence\Repository\DbalAttributeRepository
+        public: true
```

Comment:
Is it becoming public on purpose? Seems not needed but might be overlooking sth

Reply:
Remaining from testing another approach. not necessary any more